fhir-server/src/main/java/au/csiro/pathling/fhirpath/element/DatePath.java

```diff
@@ -42,20 +43,12 @@
     YEAR_ONLY_DATE_FORMAT.setTimeZone(DateTimePath.getTimeZone());
   }
 
-  /**
-   * @param expression The FHIRPath representation of this path
-   * @param dataset A {@link Dataset} that can be used to evaluate this path against data
-   * @param idColumn A {@link Column} within the dataset containing the identity of the subject
-   * resource
-   * @param valueColumn A {@link Column} within the dataset containing the values of the nodes
-   * @param singular An indicator of whether this path represents a single-valued collection
-   * @param fhirType The FHIR datatype for this path, note that there can be more than one FHIR
-   * type
-   */
-  public DatePath(@Nonnull final String expression, @Nonnull final Dataset<Row> dataset,
+  protected DatePath(@Nonnull final String expression, @Nonnull final Dataset<Row> dataset,
```

Comment:
Maybe protected constructors for abstract classes should have javadocs documentation? (that applies to all XXXPath)

Reply:
Let's aim to get the public APIs documented at this time - documenting more than that is a lot of work!